TombEngine/Renderer/Renderer11DrawEffect.cpp

```diff
@@ -65,6 +66,8 @@ namespace TEN::Renderer
 	using namespace TEN::Effects::Footprints;
 	using std::vector;
 
+	constexpr auto MAX_LIGHTNING_RANGE = 24576;
```

Comment:
Please demagic this using `BLOCK` macro.

Reply:
fixed